java/README.md

```diff
@@ -0,0 +1,17 @@
+## Tunnels Java SDK
+
+### Setting up development
+These instructions assume you are using vscode for development as SDK is configured for it.
+1. Clone this repo and open the `java` folder in vscode.
+1. Install the recommended extension pack "Extension Pack for Java"
+1. The extension will prompt you to install a JDK. Choose JDK version 11 (LTS).
+1. Once you have the extension and JDK installed, run `mvn test`.
```

Comment:
Bullet point numbers are currently all listed as `1.`

Reply:
I assume that's intentional. Markdown still displays them as an ordered list. For longer lists it can be convenient because you don't have to renumber everything when adding/removing items.

Reply:
yes, it's intentional, but then I numbered them `1. 2. 3. 4.` in the other section. I'll update it to be consistent.